apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -807,13 +759,6 @@
               }
               this.$root.$emit(AppEvent.newTab)
             })
-
-
-            if(!codeMirrorVimInstance) {
-              console.error("Could not find code mirror vim instance");
-            } else {
-              setKeybindingsFromVimrc(codeMirrorVimInstance);
-            }
           }
 
           this.editor.on("paste", (_cm, e) => {
```

Comment:
Again, all this stuff should no longer be in `TabQueryEditor`, and we shouldn't be calling methods on the child component